moped-database/views/exploded_component_arcgis_online_view.sql

```diff
@@ -1,4 +1,4 @@
--- Most recent migration: moped-database/migrations/1729197757694_remove_funding_source/up.sql
+-- Most recent migration: moped-database/migrations/1729197757695_add_agol_geometry_type/up.sql
```

Comment:
why does this needs to be dropped and recreated if nothing else was changed?

Reply:
great question and it is because this view selects from `component_arcgis_online_view` so it depends on it.

the full chain is `project_list_view` -> `component_arcgis_online_view` -> `exploded_component_arcgis_online_view` so these diffs can definitely get big when editing some of these views!!